### DISCUSSION TITLE
Help - Cannot connect to supabase project remotely

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#8421] [Questions]
Hi,

Been facing this issue recently. Every time i try to link the project from CLI:

supabase link --project-ref <project-ref>

getting this error:
Error: failed to connect to `host=db.<project-ref>.supabase.co user=postgres database=postgres`: dial error (dial tcp 3.81.166.187:5432: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.)

This is isolated to only this project. Others are working fine. But same error is happening when i try to connect via pgadmin, dbeaver, and any other ways remotely. Can someone please help?

Thanks a ton. You'll be a lifesaver

**Comment by kylewelsby** [comment]
I'm also having this error. 

```
supabase link --project-ref REDACTED
Enter your database password: 
Error: failed to connect to `host=db.<project-ref>.supabase.co user=postgres database=postgres`: dial error (timeout: dial tcp 54.75.55.175:6543: connect: operation timed out)
```

**Reply by sweatybridge** [reply]
Hello, may I check which cli version and host OS you are using? We have added [5432 as a fallback port](https://github.com/supabase/cli/blob/main/internal/db/remote/commit/commit.go#L463) so upgrading cli to latest version should resolve this issue.

**Reply by kylewelsby** [reply]
Currently using `1.7.3` upgraded to `1.7.7` resolved the issue. 

Now I'm informed my Postgresql is out-of date, but this is a different issue. 

Thanks for helping

```
Error: Remote database Postgres version 12 is incompatible with db.major_version 14. If you are setting up a fresh Supabase CLI project, try changing db.major_version in supabase/config.toml to 12.
```

**Comment by rbkayz** [comment]
If you enter your password wrong 2 times, supabase automatically locks your ip. Reach out to support to get it unblocked please

**Reply by kylewelsby** [reply]
Contacted support and theres no early indication of any banned IP addresses on my project, but I was made aware of an issue with `pgbouncer` on port `6543`.
